src/server/node_manager.c

```diff
@@ -346,15 +346,9 @@ reply_hellosvr(int stream, int need_inv)
 	if ((ret = diswsi(stream, need_inv)) != DIS_SUCCESS)
 		return ret;
 
-	/* write no of servers first as we do not write clusteraddr in case of msvr_mode. */
-	if ((ret = diswsi(stream, get_num_servers())) != DIS_SUCCESS)
```

Comment:
You need to change the mom side protocol as well in mom_server.c

Reply:
good catch, thanks